paying_for_college/tests/test_commands.py

```diff
@@ -1,33 +1,40 @@
-import mock
-import unittest
+import six
 
-from django.core.management.base import CommandError
 from django.core.management import call_command
 
 
+if six.PY2:
+    import unittest
+    import mock
+else:
+    import unittest
+    from unittest import mock
+
+
 class CommandTests(unittest.TestCase):
     def setUp(self):
         stdout_patch = mock.patch('sys.stdout')
         stdout_patch.start()
         self.addCleanup(stdout_patch.stop)
 
-    @mock.patch('paying_for_college.management.commands.'
-                'update_pfc_national_stats.nat_stats.'
-                'update_national_stats_file')
+    @mock.patch(
+        'paying_for_college.management.commands.'
+        'update_pfc_national_stats.nat_stats.'
+        'update_national_stats_file')
```

Comment:
Maybe something for broader BEWD discussion, but I feel like I'm okay with keeping long paths like these whole and using `# noqa`. Doesn't mean you have to change these.

Reply:
I felt that way at first, but then, for patches with long paths, I found it easier to track.  
I'd be happy either way.